src/pybind/mgr/influx/module.py

```diff
@@ -155,6 +157,11 @@ def init_module_config(self):
         self.config['interval'] = \
             int(self.get_config("interval",
                                 default=self.config_keys['interval']))
+        ssl = self.get_config("ssl", default=self.config_keys['ssl'])
+        self.config['ssl'] = ssl.lower() == 'true'
+        verify_ssl = \
+            self.get_config("verify_ssl", default=self.config_keys['verify_ssl'])
+        self.config['verify_ssl'] = verify_ssl.lower() == 'true'
```

Comment:
I see you cast from String to Bool here, but how does that work when somebody changes the configuration on runtime?

Reply:
I did not recognized this dynamic option. I will fix this.
Do we want `verify_ssl` to be true by default?

Reply:
@wido